android/src/main/java/com/rnfingerprint/FingerprintDialog.java

```diff
@@ -172,7 +179,7 @@ public void onAuthenticated() {
 
     @Override
     public void onError(String errorString, int errorCode) {
-        this.mFingerprintError.setText(errorString);
+        this.mFingerprintError.setText("Try again");
```

Comment:
this should coming from a strings file

Reply:
sure i will update. Thanks @rolandolopez